{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/jschaf/pylint-flask",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "GPL 2",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pylint-flask/pylint-flask-0.3.tar.gz",
    "subdirs": [
      "osx-64"
    ],
    "summary": "pylint-flask is a Pylint plugin to aid Pylint in recognizing and understanding errors caused when using Flask",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "0.3"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003"
  ],
  "package": "pylint-flask-0.3-py34_0.tar.bz2",
  "repodata": {
    "build": "py34_0",
    "build_number": 0,
    "depends": [
      "astroid >=1.0",
      "logilab-common >=0.60.0",
      "pylint >=1.0",
      "pylint-plugin-utils >=0.2.1",
      "python 3.4*"
    ],
    "license": "GPL 2",
    "md5": "96709a6e4f13ff0a652caba0d8177f0b",
    "name": "pylint-flask",
    "sha256": "8ac1705260d1d57cbbc35e55f7c5e7689ad1272add2c1640d1ff35fde8ece451",
    "size": 4486,
    "subdir": "osx-64",
    "version": "0.3"
  },
  "repodata_version": 1,
  "subdir": "osx-64",
  "url": "https://conda.anaconda.org/conda-forge/osx-64/pylint-flask-0.3-py34_0.tar.bz2"
}